src/applications/claims-status/components/ClaimPhase.jsx

```diff
@@ -179,9 +179,12 @@ export default class ClaimPhase extends React.Component {
       ) : null;
 
     return (
+      // eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions
```

Comment:
Should we have an inner element here that is a semantically clickable element? 

Also, did we intend to remove the role attribute?

Reply:
I think an inner element should be made clickable instead of the `li`, but I thought it was out of the scope of this PR. I did remove the role because "presentation" breaks the "list item" role that is auto-assigned to `li`'s and it was causing an axe error.

Reply:
Maybe for a followup ticket? 😃